src/features/callAssignments/apiTypes.ts

```diff
@@ -1,11 +1,27 @@
 export interface CallAssignmentData {
   id: number;
+  target: CallAssignmentTargetData;
```

Comment:
This is a Smart Search query. I would assume there is already a type for that somewhere, so no need to create a new custom one?